src/main/java/com/sahaya/sahayaservices/repository/TicketRepository.java

```diff
@@ -0,0 +1,15 @@
+package com.sahaya.sahayaservices.repository;
+
+import com.sahaya.sahayaservices.entity.Ticket;
+import com.sahaya.sahayaservices.enums.TicketStatus;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+
+@Repository
+public interface TicketRepository extends JpaRepository<Ticket, Long> {
+    List<Ticket> findTicketByTicketStatus(TicketStatus status);
```

Comment:
@angeleenajohn use 
``` List<Ticket> findTicketsByTicketStatusIsNot(TicketStatus status); ```
and pass status as Closed so you will get all other ticket status.